vllm/config.py

```diff
@@ -2966,6 +2972,7 @@ class CompilationConfig(BaseModel):
         sufficient for most cases. It might be beneficial to compile for
         certain small batchsizes, where inductor is good at optimizing.
     """ # noqa
+    use_v1: bool = False
     level: int = 0
     debug_dump_path: str = ""
     cache_dir: str = ""
```

Comment:
Here you can do `splitting_ops: List[str] = Field(default_factory=list)`

Reply:
We should do this in another PR